infra/routes/label_evaluator_viz_cache/spark_cache_gen.py

```
@@ -0,0 +1,263 @@
+from pathlib import Path
+
+sys.path.insert(
+    0,
+    "/Users/tnorlund/portfolio_agent_evaluator_visualization/Portfolio/receipt_dynamo",
+)
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col
+from receipt_dynamo import DynamoClient
+
+PARQUET_DIR = "/tmp/cache-dev/parquet"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/cache-dev/parquet"

(S108)